apps/andi/lib/andi/application.ex

```diff
@@ -7,7 +7,8 @@ defmodule Andi.Application do
     children = [
       AndiWeb.Endpoint,
       {Brook, Application.get_env(:andi, :brook)},
-      Andi.DatasetCache
+      Andi.DatasetCache,
+      Andi.Migration.Migrations
```

Comment:
Do you want this to happen AFTER the cache is established?  Probably doesnt matter but just asking the question

Reply:
We want the app to start up first and then run all migrations in its own time.